src/rdb/rdb_raft.c

```diff
@@ -2356,6 +2378,24 @@ rdb_raft_get_compact_thres(void)
 	return i;
 }
 
+static unsigned int
+rdb_raft_get_ae_max_entries(void)
+{
+	unsigned int n = 32;
+
+	d_getenv_int("RDB_AE_MAX_ENTRIES", &n);
+	return n;
+}
+
+static size_t
+rdb_raft_get_ae_max_size(void)
+{
+	uint64_t size = (1ULL << 20);
+
+	d_getenv_uint64_t("RDB_AE_MAX_SIZE", &size);
```

Comment:
I guess sanity checking of this value is less important since rdb_raft_clone_ae() once it gets into the per-entry loop will ensure that i=0 will transfer whatever size it has, regardless of the size limit determined here.

Reply:
Done.